samples/system-test/MicrophoneStream.test.js

```diff
@@ -1,35 +0,0 @@
-// Copyright 2016 Google LLC
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//      http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-'use strict';
-
-const path = require('path');
-const {assert} = require('chai');
-const {describe, it} = require('mocha');
-const cp = require('child_process');
-
-const execSync = cmd => cp.execSync(cmd, {encoding: 'utf-8'});
-
-const cmd = 'node MicrophoneStream.js';
-const cwd = path.join(__dirname, '..');
-
-describe('MicrophoneStream', () => {
-  it('should load and display Yaaaarghs(!) correctly', async () => {
-    const stdout = execSync(`${cmd} --help`, {cwd});
```

Comment:
For reviewers - the only thing this test did was check that the help text from yargs worked.  The actual sample requires `sox`, which is a native OS install, and a microphone.  In other words - very, very hard to test for real.